maf/src/main/java/org/mskcc/cbio/maf/MafRecord.java

```diff
@@ -103,16 +103,14 @@ public class MafRecord {
     private String maLinkMsa;
     private String maLinkPdb;
 
-    // Oncotator cols
```

Comment:
should we leave this entire `MafRecord` file as is? Doesn't feel like the oncotator -> maf renaming adds much. Prolly better to leave this for the ticket where we fix the maf parser

Reply:
but if it's easier to leave as is, that's fine too